config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   get "/healthcheck", to: proc { [200, {}, ["OK"]] }
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  get "/check-link", to: proc { [200, {}, [{}.to_json]] }
```

Comment:
I think we should stick to `uri` as much as possible internally, but we can expose them to the end user as links. At the very least, we save one byte for each check using `uri` over `link`. 😉